Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -295,13 +295,30 @@ GDCMImageIO::Read(void * pointer)
 #endif
   SizeValueType len = image.GetBufferLength();
 
+  // Decompress the Pixel Data buffer when needed. This is required here in the
+  // pipeline to make sure to decompress JPEGBaseline1 into YBR_FULL.
```

Comment:
This comment is misleading, it is not required for color files at all.
The block is required only for relatively rare monochrome1, but will run on all encapsulated files. At least useless overheat and possibly new bugs.
Sorry for writing, but is related to color images stuff where i was involved into.

Reply:
Correct, the comment above could be rephrased, I'll submit a PR.

Reply:
We could put the new phrasing in this PR, if you wish. Give me the updated text and I will update the PR. You could also pull this PR, update it and push it to your original PR #1350.